app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/EditServiceMonitoringTest.php

```diff
@@ -81,10 +82,19 @@
         ->call('save')
         ->assertHasNoFormErrors();
 
+    /** @var string $description */
+    $description = $serviceMonitoringTarget->fresh()->description;
+
+    /** @var string $domain */
+    $domain = $serviceMonitoringTarget->fresh()->domain;
+
+    /** @var ServiceMonitoringFrequency $frequency */
+    $frequency = $serviceMonitoringTarget->fresh()->frequency;
```

Comment:
Ah, we should not have done this before. Rather than calling `fresh()` every time here, before all these variable definitions lets just do one `$serviceMonitoringTarget->refresh()`.

Which will bring these 3 queries, 2 of which are not needed, down to 1.